app/dashboard/templates/shared/onepager_auth.html

```diff
@@ -17,11 +17,11 @@
 {% load i18n static %}
 {% load is_in_list %}
 
-<link rel="stylesheet" href="{% static "v2/css/fontawesome-all.min.css" %}">
```

Comment:
is this by design?

Reply:
dead code I need to remove that we already loading the header now in tips pages and the footer_scripts